website/docs/patterns/description-list-patterns/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Description list patterns
+description: Guidelines and best practices for creating description lists with Helios Foundations.
+caption: Guidelines and best practices for creating description lists with Helios Foundations.
+related: ['foundations/typography', 'foundations/colors']
```

Comment:
[nit] This related content doesn't seem _too_ related to this pattern because these foundations are only _used_ in the pattern (which you could say the same about anything), though, I'm not actually sure what I would put here, so feel free to disregard this.